src/main/java/bio/terra/workspace/service/workspace/WorkspaceService.java

```diff
@@ -31,33 +31,33 @@ public WorkspaceService(JobService jobService, WorkspaceDao workspaceDao, SamSer
   }
 
   @Traced
-  public CreatedWorkspace createWorkspace(
-      CreateWorkspaceRequestBody body,
+  public UUID createWorkspace(
```

Comment:
add javadoc for this method. At least saying what is being returned since `UUID` isn't very informative.